base/build.gradle.kts

```diff
@@ -33,6 +33,11 @@ sourceSets.main {
 
 tasks.compileJava { dependsOn(generateVersion) }
 tasks.sourcesJar { dependsOn(generateVersion) }
+tasks.withType<org.aya.gradle.GenerateReflectionConfigTask>().configureEach {
+  extraDir = buildDir.resolve("classes/java/main/org/aya/core/serde")
+  classPrefixes = listOf("SerTerm$", "SerPat$", "SerDef$")
+  packageName = "org.aya.core.serde"
```

Comment:
What would happen if `listOf("SerTerm", "SerPat", "SerDef")` so everything under `org.aya.core.serde` can be generated?

Reply:
I don't know

Reply:
I don't understand the sentence

Reply:
I will take this. Don't worry 😄